### PR TITLE
new repo - cluster-api-actuator-pkg

### PR DESCRIPTION
**Pull request by paulfantom**
Adding CI pipeline for new repository. 

This repository shouldn't create any container images.

/assign @stevekuznetsov @petr-muller

### CODE REVIEW

File: cluster/ci/config/prow/plugins.yaml [hunk 1]
```diff
@@ -291,6 +291,7 @@ approve:
   - openshift/machine-api-operator
   - openshift/cluster-api-provider-aws
   - openshift/cluster-api-provider-libvirt
+  - openshift/cluster-api-actuator-pkg
```
[the remark this answers is not in the document]
Comment (paulfantom) [reply cluster/ci/config/prow/plugins.yaml]: Done

**Comment by paulfantom** [comment]
All fixed.

File: ci-operator/templates/openshift/cluster-api-provider-libvirt/cluster-launch-packet.yaml [hunk 2]
```diff
@@ -0,0 +1,182 @@
+kind: Template
```
Comment (stevekuznetsov) [inline ci-operator/templates/openshift/cluster-api-provider-libvirt/cluster-launch-packet.yaml]: Is this used?

**Pull request merged**